src/css/components/nav.scss

```diff
@@ -37,16 +37,23 @@
     &:hover {
       color: $color-primary;
     }
+    &.external-link {
```

Comment:
Since this is nested within an `a`, and uses the `&`, this also throws a qualifying type error in the linter, since the css resolves to `a.external-link`. I would put the `&.external-link` declaration directly inside the `nav-link` body.